### DISCUSSION TITLE
Error when execute **npx hardhat compile** in hardhat-starter-kit

### DISCUSSION THREAD

**Discussion by eludius18** [smartcontractkit/full-blockchain-solidity-course-py#273] [Q&A] [answered]
Hi Everyone,

I just found an error when execute **npx hardhat compile** in hardhat-starter-kit :

An unexpected error occurred:

Error: Cannot find module '@nomiclabs/hardhat-waffle'

I tried to install '@nomiclabs/hardhat-waffle' and the other requirements in hardhat-config.js but remain not running...

**Comment by cromewar** [comment]
Did you install the module using yarn or npm? I had a similar problem installing with yarn, could you try and install it using npm?

I also found a similar problem here:
[Error: Cannot find module '@nomiclabs/hardhat-waffle'](https://issueexplorer.com/issue/nomiclabs/hardhat/1890)

However as this is outside the scope of this course, could you give me a hand on this one pal @PatrickAlphaC ?

**Reply by muhammadhamzahabibhashmi** [reply]
It requires the  '@nomiclabs/hardhat-waffle' 
Try "npm install -D @nomiclabs/hardhat-waffle ethereum-waffle"

**Comment by PatrickAlphaC** [comment, marked as answer]
Hello! Could you actually make this issue on the [hardhat starter kit](https://github.com/smartcontractkit/hardhat-starter-kit)? Thank you!
